src/DDoSX/CdnClient.php

```diff
@@ -23,14 +30,59 @@ public function create(Cdn $cdn)
     {
         $response = $this->post(
             'v1/domains/' . $cdn->name . '/cdn',
-            json_encode($this->friendlyToApi($cdn, $this->requestMap))
+            json_encode($this->friendlyToApi($cdn, self::CDN_MAP))
         );
         $body = $this->decodeJson($response->getBody()->getContents());
 
         return (new SelfResponse($body, "domain_name"))
             ->setClient($this)
             ->serializeWith(function ($response) {
-                return new Cdn($this->apiToFriendly($response->data, $this->requestMap));
+                return new Cdn($this->apiToFriendly($response->data, self::CDN_MAP));
             });
     }
+
+    /**
+     * Get a paginated list of cdn rules for a domain
+     * @param $domainName
+     * @param int $page
+     * @param int $perPage
+     * @param array $filters
+     * @return int|\UKFast\SDK\Page
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getRules($domainName, $page = 1, $perPage = 15, $filters = [])
+    {
+        $page = $this->paginatedRequest(
+            'v1/domains/' . $domainName . '/cdn/rules',
+            $page,
+            $perPage,
+            $filters
+        );
+
+        $page->serializeWith(function ($item) {
+            return $this->serializeCdnRule($item);
+        });
+
+        return $page;
+    }
+
+    /**
+     * Get a rule for a domain by it's ID
```

Comment:
```suggestion
     * Get a rule for a domain by its ID
```


FYI
> **Its** is a possessive form of the pronoun it, meaning belonging to it.
> **It’s** is a contraction of the words it is or it has. (Interestingly, we don’t really contract it was into it’s.)

https://www.dictionary.com/e/its-vs-its/